<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>- Foto Sello -</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jmogan.png</normaloff>jmogan.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOriginales">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>8</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No se ha seleccionado ninguna ruta</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDestino">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>68</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No se ha seleccionado ninguna ruta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>238</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(204, 0, 0)</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="carpetaDestino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>68</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Carpeta Destino</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progreso">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="inf_izq">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>112</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Inferior izq.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pegarSello">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>238</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(115, 210, 22)</string>
    </property>
    <property name="text">
     <string>Pegar Sello</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seleccionarFotos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar fotos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seleccionarSello">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>128</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar sello</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="inf_centro">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inferior centro</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSello">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>No se ha seleccionado ninguna ruta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
